content/docs/2.10/scalers/prometheus.md

```diff
@@ -36,6 +37,7 @@ triggers:
 - `activationThreshold` - Target value for activating the scaler. Learn more about activation [here](./../concepts/scaling-deployments.md#activating-and-scaling-thresholds).(Default: `0`, Optional, This value can be a float)
 - `namespace` - A namespace that should be used for namespaced queries. These are required by some highly available Prometheus setups, such as [Thanos](https://thanos.io). (Optional)
 - `cortexOrgID` - The `X-Scope-OrgID` header to query multi tenant [Cortex](https://cortexmetrics.io/) or [Mimir](https://grafana.com/oss/mimir/). (Optional)
+- `customHeaders` - Custom headers to include while querying the prometheus endpoint.
```

Comment:
Might we worth noting here, that for auth related headers one should use Custom Authentication mentioned below.

Reply:
@zroubalik updated